mapproxy/request/wms/__init__.py

```diff
@@ -349,7 +371,7 @@ class WMS130MapRequest(WMSMapRequest):
     request_params = WMS130MapRequestParams
     xml_exception_handler = exception.WMS130ExceptionHandler
     fixed_params = {'request': 'GetMap', 'version': '1.3.0', 'service': 'WMS'}
-    expected_param = ['version', 'request', 'layers', 'styles', 'crs', 'bbox',
```

Comment:
Why have you removed those parameters?

Reply:
They were needed for a previous dynamic version of the patch that ended up being scraped